frontend/src/scenes/experiments/MetricsView/DeltaChart.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { ExperimentMetric, ExperimentMetricType, NodeKind } from '~/queries/schema/schema-general'
+
+import { getDefaultMetricTitle } from './DeltaChart'
+
+describe('getDefaultMetricTitle', () => {
+    it('handles ExperimentEventMetricConfig correctly', () => {
+        const metric: ExperimentMetric = {
+            kind: NodeKind.ExperimentMetric,
+            metric_type: ExperimentMetricType.COUNT,
+            metric_config: {
+                kind: NodeKind.ExperimentEventMetricConfig,
+                math: 'total',
+                math_property: undefined,
+                event: 'purchase completed',
+            },
+        }
+
+        expect(getDefaultMetricTitle(metric)).toBe('Total purchase completed')
```

Comment:
I find this a little difficult to read, to be honest.

What about using the raw event name as the fallback title, and then displaying the aggregation type as the badge?

![CleanShot 2025-02-18 at 06 58 08@2x](https://github.com/user-attachments/assets/9fc49337-c957-4dad-83a7-795aad715503)

Reply:
Yeah, good point. Didn't consider leveraging the badge here. I'll iterate a bit more on this.

Reply:
Fixed in 6d0dcb84c30423dc7ed3ec95ae2d49c55ea012cd